sdk/servicebus/service-bus/src/receiver.ts

```diff
@@ -341,8 +343,7 @@ export class SessionReceiver {
   }
 
   /**
-   * Renews the lock on the session.
-   * Check the `sessionLockedUntilUtc` property on the reciever for the time when the lock expires.
+   * Renews the lock on the session to be set to 30 seconds from the moment of invocation.
```

Comment:
Is this 30 seconds configurable just like for message lock duration for non sessions? Or is it set in the sdk?

Look at the documentation we have for `renewMessageLock`. We talk about the lock duration set during Queue creation.

#2059 was about if the same applies for session locks. If yes, then the docs for `renewSessionLock` needs to be updated just like `renewMessageLock`

Reply:
For session lock, it's being returned by the Service Bus and not set in the SDK.

Copy-pasting comment here, in case it was missed -

"Ran tests and verified that Service Bus returns 30 seconds for both initial and extension duration periods.
Also, this value is available only after _init() is invoked and underlying messageSession is defined."

Reply:
The copy-pasted comment was not missed. I did read it.
I understand that after every invocation of `renewSessionLock`, the lock is extended for 30 more seconds.

The question is, whether the user can configure this duration of 30 seconds in any way.

The documentation in this PR for `renewSessionLock` seem to suggest that the user cannot configure this 30 seconds duration. 

The copy-pasted comment doesnt mention whether it was investigated that this 30 seconds duration indeed is not configurable.


The message lock duration is configurable when the Queue is being created in the portal. 

- Is there a way to configure the lock duration for the sessions in the same way? 
- Or does the same value (for message lock duration as configured in the portal) is being used for the session lock duration

Reply:
Oh okay, I had configured the lock duration on queue to be for 60 seconds, and this had no effect on session lock duration.
I had looked up documentation online on Service Bus, ran into [this](https://social.msdn.microsoft.com/Forums/vstudio/en-US/fe1ee7bb-cc17-4852-a577-481b520f58c9/lock-duration-on-queues-with-session-enabled?forum=servbus) but couldn't locate any documentation on where to set the duration. 
Based on code, Service Bus returns this value and it's not clear where it picks from. (Definitely not the lock duration set on queue).
(Sorry I thought I already surfaced this investigation bit elsewhere!)